Avoid Runtime Error during file open, especially when there IS a legal
check if it succeded or not

--- fpcbuild_2.0.4_exp/fpcsrc/rtl/linux/linuxvcs.pp~	2006-02-15 11:04:09.000000000 +0100
+++ fpcbuild_2.0.4_exp/fpcsrc/rtl/linux/linuxvcs.pp	2006-10-23 12:52:49.052613239 +0200
@@ -94,8 +94,10 @@
   pid:=fpgetpid;
   repeat
     str(pid,s);
+    {$I-}
     assign(f,'/proc/'+s+'/stat');
     reset(f);
+    {$I+}
     if ioresult<>0 then
       break;
     read(f,dummy);
--- fpcbuild_2.0.4_exp/fpcsrc/utils/grab_vcsa.pp~	2006-02-15 11:04:09.000000000 +0100
+++ fpcbuild_2.0.4_exp/fpcsrc/utils/grab_vcsa.pp	2006-10-23 12:52:19.823019988 +0200
@@ -78,8 +78,10 @@
       pid:=fpgetpid;
       repeat
         str(pid,s);
+	{$I-}
         assign(f,'/proc/'+s+'/stat');
         reset(f);
+	{$I+}
         if ioresult<>0 then
           begin
             found_vcsa:=false;
